app/src/main/java/com/github/geohunt/app/ui/screens/main/MainViewModel.kt

```diff
@@ -41,6 +50,32 @@ class MainViewModel(
         }
     }
 
+    fun listenForNewNotification(context: Context) {
+        val database = FirebaseDatabase.getInstance()
```

Comment:
Shouldn't this be put in a repository?

Reply:
I mean, it's fine but this should be tested as well, no?

Reply:
Apparently testing whether notifications show up is very hard and outside the scope of unit test. I searched ways and I'll have to install other libraries and incorporate end to end tests but it's kinda hard